app/services/oidc_jwt_validate.rb

```diff
@@ -12,8 +12,8 @@ def decoded_payload
         verify_aud: true,
         iss: Setting['oidc_issuer'],
         verify_iss: true,
-        algorithms: [Setting['oidc_algorithm']],
-        jwks: jwks_loader }
+        algorithms: [Setting['oidc_algorithm']] },
+      jwk_findkey
```

Comment:
Don't we need to have (I'm not sure, because tests fail because of the `lambda` syntax, but I think so):
```suggestion
      &jwk_findkey
```